controllers/overcloudipset_controller.go

```diff
@@ -106,12 +107,13 @@ func (r *OvercloudIPSetReconciler) Reconcile(req ctrl.Request) (ctrl.Result, err
 			if err != nil {
 				if k8s_errors.IsNotFound(err) {
 					r.Log.Info(fmt.Sprintf("OvercloudNet named %s not found!", netName))
-					return ctrl.Result{}, nil
 				}
 				// Error reading the object - requeue the request.
 				return ctrl.Result{}, err
 			}
 
+			instance.Status.Networks[network.Name] = network.Spec
```

Comment:
Why do we store an entire `OvercloudNet` spec here instead of just its name?  Just curious.

Reply:
So the 'Spec' has the information we need in it, namely the gateway. I thought about creating a duplicate struct for this but the struct has what we need and as this is just a cache it seemed reasonable.